tests/cypress/integration/actions_tasks_objects/case_24_delete_unlock_lock_object.js

```diff
@@ -0,0 +1,129 @@
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+import { taskName, labelName } from '../../support/const';
+
+context('Delete unlock/lock object', () => {
+    const caseId = '24';
+
+    const createRectangleShape2Points = {
+        points: 'By 2 Points',
+        type: 'Shape',
+        labelName: labelName,
+        firstX: 100,
+        firstY: 100,
+        secondX: 300,
+        secondY: 300,
+    };
+
+    function lockObject() {
+        cy.get('.cvat-objects-sidebar-state-item').within(() => {
+            cy.get('.cvat-object-item-button-lock').click();
+        });
+    };
+
+    function deleteObjectViaShortcut(shortcut, stateLockObject) {
+        if (stateLockObject == 'unlock') {
+            cy.get('.cvat-canvas-container').within(() => {
+                cy.get('.cvat_canvas_shape')
+                    .trigger('mousemove')
+                    .should('have.class', 'cvat_canvas_shape_activated');
+            });
+        };
+        cy.get('body')
+            .type(shortcut);
+    };
+
+    function clickRemoveOnDropdownMenu() {
+        cy.get('.ant-dropdown')
```

Comment:
``.ant-dropdown`` is too abstract. Please, use another selector. You can add any css classes to the application (i.e. ``.cvat-object-item-menu``)

Reply:
All cssSelectors replaced from "ant*" to "cvat*"